v1/README.md

```diff
@@ -1,4 +0,0 @@
-# P4Runtime Specification Version 1
-
-This directory contains protobuf files, specifications and related artifacts for
```

Comment:
Should we modify this text in the top-level README and possibly add some READMEs into the `proto` directory structure at suitable places?
> This directory contains protobuf files, specifications and related artifacts for all versions of the P4Runtime API. A separate directory is assigned to each major version, e.g. v1. Therein, tags are used to mark minor and patch release versions.

Reply:
Thanks for pointing this out, I'm going to update the README before I merge